tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Cumulative_Interactions.yaml

```diff
@@ -2,7 +2,7 @@ _file_name: Cumulative_Interactions.yaml
 cache_timeout: null
 certification_details: null
 certified_by: null
-dataset_uuid: 1b1cbf0a-1193-4251-ad52-724c2f0190ae
+dataset_uuid: 649ad1d3-c0b4-48e1-a02b-e1d1e80c94ac
```

Comment:
This changes the dataset from `fact_navigation_completion` to `fact_at_risk_navigation_completion`, which is wrong for the course dashboard.